py-rse/py-scripting.Rmd

```diff
@@ -245,66 +360,74 @@ from collections import Counter
 def count_words(reader):
     """Count the occurrence of each word in a string."""
     text = reader.read()
-    findwords = re.compile(r"\w+", re.IGNORECASE)
+    findwords = re.compile(r"\w+")
```

Comment:
You've removed `re.IGNORECASE` which I can see the logic for (one less thing to explain), but it still appears in versions of `countwords.py` that appear below.

Reply:
I've taken out the regexp stuff completely - please check.